Running make build fails on latest

First of all, thanks for creating this! It solves a real problem for shipping code with bundled dependencies 👍 

I saw on the readme it says that the phar is coming soon, but by the looks of things it seems as though it may be possible to do yourself? I tried building the phar with `make build` but it's looking for `box` which is missing in the `vendor-bin` directory.

```
make build
# everything goes fine until this part...
composer dump-autoload -d build  --classmap-authoritative
Generating optimized autoload files
php -d zend.enable_gc=0 vendor-bin/box/vendor/bin/box build
Could not open input file: vendor-bin/box/vendor/bin/box
make: *** [build] Error 1
```

Am I missing something?

--- a/.travis.yml
+++ b/.travis.yml
@@ -25,10 +25,6 @@ before_script: phpenv config-rm xdebug.ini || true
 install:
   - set -eo pipefail
   - composer install --no-interaction --no-progress --no-suggest --prefer-dist $COMPOSER_FLAGS
-  - |
-    if [ "$COVERAGE" == "true" ]; then
-        composer bin box install --no-interaction --no-progress --no-suggest --prefer-dist $COMPOSER_FLAGS
-    fi
 
 script:
   - |

--- a/Makefile
+++ b/Makefile
@@ -15,7 +15,7 @@ help:
 ##---------------------------------------------------------------------------
 
 build:		## Build the PHAR
-build: vendor
+build: vendor vendor-bin/box/vendor
 	# Cleanup existing artefacts
 	rm -f bin/php-scoper.phar
 	rm -rf build
@@ -78,10 +78,16 @@ tb: vendor
 vendor: composer.lock
 	composer install
 
+vendor-bin/box/vendor: vendor-bin/box/composer.lock
+	composer bin all install
+
 fixtures/set005/vendor: fixtures/set005/composer.lock
 	 composer -d=fixtures/set005 install
 
 composer.lock: composer.json
+	@echo compose.lock is not up to date.
+
+vendor-bin/box/composer.lock: vendor-bin/box/composer.json
 	@echo compose.lock is not up to date.
 
 bin/scoper.phar: bin/php-scoper src vendor scoper.inc.php

